package.json

```diff
@@ -30,10 +30,7 @@
   },
   "dependencies": {
     "app2png": "^1.0.0",
-    "async": "^2.1.4",
     "fuzzyfind": "^2.0.0",
-    "iconutil": "^1.0.2",
-    "mkdirp": "^0.5.1",
-    "simple-plist": "^0.2.1"
```

Comment:
It uses mdfind to get the `CFBundleDisplayName`.

https://github.com/tinytacoteam/zazu-file-finder/blob/master/adapters/osx-mdfind/index.js#L77-L78